index.html

```diff
@@ -16,13 +16,14 @@
 			<div class="img"><img src="assets/images/avatar-jessica.jpeg" alt="Zdjęcie profilowe" class="profile_img" width="90px" height="90px" /></div>
 
 			<div class="details">
-				<h1>Jessica Randall</h1>
-				<h2>London, United Kingdom</h2>
+				<h1 class="initials">Jessica Randall</h1>
+				<h2 class="location">London, United Kingdom</h2>
 			</div>
+      
 			<div class="quote">
 				<p>"Front-end developer and avid reader."</p>
 			</div>
```

Comment:
```suggestion
	<p class="quote">"Front-end developer and avid reader."</p>
```

Nie potrzebujemy tutaj diva, wystarczy samo p, które i tak zostanie oddzielone gapem przed flexa w `.container`, a dodatkowo pozbędziemy się jednego znacznika div ze struktury html

Reply:
Done